bekeplar/EPIC-Mail
# Ft fetch user create a group 165084341

## What does this PR do?
- Adds feature fetch user able to create a group and own it.
## Tasks completed
- writing JS code to link frontend and Backend 
- Changing some HTML and CSS files
## How should this be manually tested?
On this Pull request navigate to the files changed tab to see the various files changed
Screenshots
## Stories attached
[#165084341, fetch API user create a group.]

kindly review for some feedback
@nanfuka, @SentongoHabib, @neelxie, @alexxsanya, @Emmanuelaaron, @jennizalwango
